opentracing-shim/build.gradle.kts

```diff
@@ -11,11 +11,15 @@ dependencies {
 
   api("io.opentracing:opentracing-api")
   implementation(project(":semconv"))
+  implementation("io.opentracing:opentracing-noop:0.33.0")
```

Comment:
We centrally manage dependency version in [./dependencyManagement/build.gradle.kts](https://github.com/open-telemetry/opentelemetry-java/blob/main/dependencyManagement/build.gradle.kts#L77). Can you add this entry to the list?